test/integration/model/json.test.js

```diff
@@ -310,6 +310,47 @@ describe(Support.getTestDialectTeaser('Model'), function() {
           });
         });
       });
+
+      it('should be possible to destroy with where', function () {
+        var conditionSearch = {
+          where: {
+            data: {
+              employment : 'Hacker'
+            }
+          }
+        };
+
+        return Promise.join(
+          this.Event.create({
+            data: {
+              name: {
+                first: 'Elliot',
+                last: 'Alderson'
+              },
+              employment: 'Hacker'
+            }
+          }),
+          this.Event.create({
+            data: {
+              name: {
+                first: 'Christian',
+                last: 'Slater'
+              },
+              employment: 'Hacker'
```

Comment:
Could you give one row another employment - theoretically this test also passed if the table is truncated